packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -11,14 +11,19 @@ const CFD_text: { [key: string]: string } = {
     mt5_cfds: 'MT5 CFDs',
     cfd: 'CFDs',
     synthetic: 'Derived',
+    synthetic_demo: 'Derived Demo',
     synthetic_bvi: 'Derived BVI',
     synthetic_svg: 'Derived SVG',
     synthetic_v: 'Derived Vanuatu',
     financial: 'Financial',
+    financial_demo: 'Financial Demo',
     financial_bvi: 'Financial BVI',
     financial_fx: 'Financial Labuan',
     financial_v: 'Financial Vanuatu',
     financial_svg: 'Financial SVG',
+    all: 'Swap-Free',
```

Comment:
You can safely remove the comment at line no.7

Reply:
ok